### PR TITLE
Added doc to PSP

### PR DESCRIPTION
**Pull request by mnevadom**
Regarding this issue, already solved

https://github.com/okteto/app/issues/4994

### CODE REVIEW

**Comment by pchico83** [comment]
@mnevadom could you create a branch `release-1.2` and send the PR against it?

**Comment by mnevadom** [comment]
> @mnevadom could you create a branch `release-1.2` and send the PR against it?

Done

**Review by AgustinRamiroDiaz — APPROVED** [review]

File: src/content/self-hosted/administration/configuration.mdx [hunk 1]
```diff
@@ -717,6 +717,20 @@ networkPolicies:
   - "169.254.169.254/32"
 ```
 
+### podSecurityPolicy
+
+Decide if you want [pod security policy](https://kubernetes.io/docs/concepts/security/pod-security-policy/) to be added to the Roles and ClusterRoles
```
Comment (pchico83) [inline src/content/self-hosted/administration/configuration.mdx:722]:
```suggestion
Decide if you want [pod security policy](https://kubernetes.io/docs/concepts/security/pod-security-policy/) to be added to the service accounts created by Okteto.
```

File: src/content/self-hosted/administration/configuration.mdx [hunk 2]
```diff
@@ -717,6 +717,20 @@ networkPolicies:
...
+Enabled by default.
+
+- `useAppArmor`: Add the apparmor security annotations to the restrictive PSP
```
Comment (pchico83) [inline src/content/self-hosted/administration/configuration.mdx:725]:
```suggestion
- `useAppArmor`: Add the apparmor security annotations to the Okteto pod security policies.
```

**Pull request merged**